noaa/README.md

```diff
@@ -53,6 +53,7 @@ This track allows to overwrite the following parameters with Rally 0.8.0+ using
 * `index_settings`: A list of index settings. Index settings defined elsewhere (e.g. `number_of_replicas`) need to be overridden explicitly.
 * `cluster_health` (default: "green"): The minimum required cluster health.
 * `error_level` (default: "non-fatal"): Available for bulk operations only to specify ignore-response-error-level.
+* `max_num_segments` (default: 1): An integer specifying the max amount of segments the force-merge operation should use. To unset, set the parameter to `null`.
```

Comment:
IMHO we should explicitly state here that this requires that track parameters are provided via a JSON file (with a link to the docs). Passing `null` via the command line parameter `--track-params` is not supported.